CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ set(XMS_BUILD NO CACHE BOOL "Set this if you want to use this package with XMS."
 set(PYTHON_TARGET_VERSION 3.10 CACHE STRING "Version of python to link to for python wrapping.")
 set(IS_PYTHON_BUILD NO CACHE BOOL "Set this if you want to build the python bindings.")
 set(XMS_TEST_PATH ${PROJECT_SOURCE_DIR}/test_files/ CACHE PATH "Path to test files for testing")
-set(XMS_VERSION "\"99.99.99\"" CACHE STRING "Library Version")
+set(XMS_VERSION "99.99.99" CACHE STRING "Library Version")
```

Comment:
Did you remove these quotes or was it a linter? Does the python package still work with it like this?

Reply:
I meant to remove the quotes. The fix is for manual cmake builds. It makes it so you don't have to edit the default XMS_VERSION each time by removing the quotes. This doesn't affect the Python package because it gets its version from _package/xms/core/__init__.py, and for tagged C++ builds XMS_VERSION gets set by the environment variable, so both will work.